package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@hawktracer/client",
+  "version": "1.0.0",
+  "description": "A library that obtains profile data generated by HawkTracer",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "repository": "https://github.com/amzn/hawktracer.git",
+  "author": "Hojin Ghim <hojin@amazon.com>",
+  "license": "MIT",
+  "scripts": {
+    "test": "jest",
+    "prepare": "tsc",
+    "install": "cmake-js compile --target nodejs_bindings_client --CDENABLE_CLIENT=YES --CDENABLE_NODEJS_BINDINGS=YES --CDBUILD_STATIC_LIB=YES"
+  },
+  "dependencies": {
+    "@types/bindings": "^1.3.0",
+    "bindings": "^1.3.0",
+    "cmake-js": "^6.1.0",
+    "node-addon-api": "https://github.com/nodejs/node-addon-api.git#7f56a78"
```

Comment:
I used this commit because it fixes https://github.com/nodejs/node-addon-api/issues/660 but it's not included in any released version yet.